tasks/build.js

```diff
@@ -0,0 +1,28 @@
+const bundleCSS = require('./bundle-css');
```

Comment:
It looks very similar to `release.js` and `debug.js` files. So I suggest you reuse the code (maybe even merge everything into a single file, use `git mv` to keep the file history) and use command line arguments. Here's an [example of running a script with arguments](https://github.com/alexanderby/malevic/blob/master/package.json#L18) and [here how they are read](https://github.com/alexanderby/malevic/blob/master/tasks/build.js#L177) in Node.js script. So, later you will be able to create multiple tasks like `release`, `debug` and `debug-watch` that will run `node tasks/build.js --release` or `node tasks/build.js --debug --watch`.

Reply:
Will have a look and merge everything into a single file if I figure out to merge to single file with preserving history as git mv isn't the best option I thought.

Reply:
You can move a single file and remove another one.

Reply:
You can't move into other files AFAIK

Reply:
Yes, you have to remove the file you created, and then move the source file into new destination with `git mv`.

Reply:
Will try